### PR TITLE
Update the changelog

### PR DESCRIPTION
**Pull request by sbrunner**
Do some other little changes found during the first migration...

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
-  diff: https://github.com/camptocamp/demo_geomapfish/compare/prod-2-5...prod-2-6#diff-49e4e3e08323fa260d0e7682f102a585bb0ee2d25f0434e47255519c0b0423a5
-
-Changes in the `geoportal/vars.yaml` file:
-  https://github.com/camptocamp/demo_geomapfish/compare/prod-2-5...prod-2-6#diff-d8128a088c1fb76a5d2e77bfc707d1426f5bd1511d430f20a724c0e4955b3db7
-
-- Optional point: the header can be in a separate file, diff in the `*.html.ejs` file:
+Client application configuration
+................................
+
+Acctually we have some configuration in the `*.html.ejs`, in the `Controler*.js`, and in the vars.yaml file.
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:13]: Previous versions had configuration in the ...

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 2]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+The goal of this change is to move all thies configurationthe the `vars.yaml` file (with default values in
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:14]: "is to move all thies configurationthe" -> "is to move this configuration to"

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 3]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+the `CONST_vars.yaml`). This will make the future upgrade easyer and incrase the configuration possibility
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:15]: "the future upgrade easyer and incrase the configuration possibility" -> "future upgrades easier and increase configuration possibilities"

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 4]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+in the simple application mode, see the documentation
+<https://camptocamp.github.io/c2cgeoportal/${MAIN_BRANCH}/integrator/create_application.html#simple-application>.
+
+
+When you apply the `ngeo.diff` you should first apply the changes in the `geoportal/vars.yaml` file.
+
+For the next you should consultate the documentation about:
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:22]: For the next step, you should consult the documentation about:

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 5]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+- The interfaces_config documentation
+<https://camptocamp.github.io/c2cgeoportal/${MAIN_BRANCH}/integrator/ngeo.html?highlight=ngeo#dynamic-json-view>
+- GMF constants definitions <https://camptocamp.github.io/ngeo/master/jsdoc/module-contribs_gmf_src_options.html>
+- ngeo constants definitions <https://camptocamp.github.io/ngeo/master/jsdoc/module-src_options.html>
+
+The 'srid', 'alternate_projections', 'resolutions' and 'extent' `add path` will be configure on one point and will be
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:28]: configure -> configured

But what do we mean with "on one point"?

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 6]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+dispatch using c2ctemplate or YAML link.
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:29]: dispatch -> dispatched

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 7]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+
+Kind of thinks you can see:
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:31]: In the following, we show typical examples of necessary changes:

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 8]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+...
+-// The URL to the search service.
+-config.searchUrl = '{FULL_ENTRY_POINT}search?interface=api&limit=15';
+```
+And in your `geoportal/cartoriviera_geoportal/static-ngeo/api/index.js` file:
+```
+// The URL to the search service.
+onfig.searchUrl = '{FULL_ENTRY_POINT}search?interface=api&limit=15';
+```
+We see that that your settings didn't corresponds to the previus default settings, than you should have in
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:49]: We see that the project settings didn't correspond to the previous default settings, therefore you should have in
your vars file something like:

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 9]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+vars:
+  interfaces_config:
+    api:
+      routes:
+        searchUrl:
+          params:
+            limit: 15
+
+update_paths:
+  - interfaces_config.api.routes.searchUrl  # This one sould already be present
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:61]: sould -> should

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 10]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+      srid: 2056,
+      mapViewConfig: {
+        center: [2559045, 1144195],
+        zoom: 1,
+        constrainResolution: true,
+        resolutions: [50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5, 0.25, 0.1, 0.05, 0.025]
+      }
+    }, $scope, $injector);
+```
+We see that that your settings didn't corresponds to the previus default settings, then you should have in
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:96]: We see that the project settings didn't correspond to the previous default settings, therefore you should have in
your vars file something like:

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 11]
```diff
@@ -7,27 +7,138 @@ Version 2.6.0
...
+
+In the ngeo.diff:
+```
+-    this.elevationLayers = ['aster', 'srtm'];
+```
+And in your `geoportal/cartoriviera_geoportal/static-ngeo/api/index.js` file:
+```
+   this.elevationLayers = ['mns', 'mnt', 'rayglobal'];
+```
+We see that that your settings didn't corresponds to the previus default settings, than you should have in
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:127]: We see that the project settings didn't correspond to the previous default settings, therefore you should have in
your vars file something like:

**Review by jwkaltz — APPROVED** [review]

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl (hunk 5 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:28]: => change the sentence ...

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py [hunk 12]
```diff
@@ -685,6 +685,7 @@ def step9(self, step):
         status = self.get_modified(
             "CONST_create_template/geoportal/{}_geoportal/static-ngeo".format(self.project["project_package"])
         )
+        status += "CONST_create_template/geoportal/vars.yaml"
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py:688]:
```suggestion
        status += ["CONST_create_template/geoportal/vars.yaml"]
```

**Review by arnaud-morvan — APPROVED** [review]
Looks good, minor comment

**Pull request merged**